feature-wallet-api/src/main/java/io/novafoundation/nova/feature_wallet_api/domain/model/BalanceHold.kt

```diff
@@ -0,0 +1,35 @@
+package io.novafoundation.nova.feature_wallet_api.domain.model
+
+import io.novafoundation.nova.common.utils.Identifiable
+import io.novafoundation.nova.core_db.model.BalanceHoldLocal
+import io.novafoundation.nova.core_db.model.BalanceHoldLocal.HoldIdLocal
+import io.novafoundation.nova.feature_wallet_api.data.network.blockhain.types.Balance
+import io.novafoundation.nova.feature_wallet_api.domain.model.BalanceHold.HoldId
+import io.novafoundation.nova.runtime.multiNetwork.chain.model.Chain
+
+class BalanceHold(
+    val id: HoldId,
+    val amountInPlanks: Balance,
+    val chainAsset: Chain.Asset
+) : Identifiable {
+
+    class HoldId(val module: String, val reason: String)
+
+    // Keep in tact with `BalanceBreakdownIds`
+    override val identifier: String = "${id.module}: ${id.reason}"
+}
+
+fun mapBalanceHoldFromLocal(
+    asset: Chain.Asset,
+    hold: BalanceHoldLocal
+): BalanceHold {
+    return BalanceHold(
+        id = hold.id.toDomain(),
+        amountInPlanks = hold.amount,
+        chainAsset = asset
+    )
+}
```

Comment:
It's not a best practice to keep data -> domain mappers in domain layer. Let's move it to the data directory?